.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,123 @@
+name: Publish release
+
+on:
+  push:
+    branches: [ prod ]
+  workflow_dispatch:
+
+jobs:
+  correct_repository:
+    runs-on: ubuntu-latest
+    steps:
+      - name: fail on fork
+        if: github.repository_owner != 'cmu-delphi'
+        run: exit 1
+
+  check:
+    runs-on: ubuntu-latest
+    needs: correct_repository
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+        with:
+          ssh-key: ${{ secrets.CMU_DELPHI_DEPLOY_MACHINE_SSH }}
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install bump2version
+        run: python -m pip install bump2version
+      - name: Check for delphi-utils changes
+        uses: dorny/paths-filter@v2
+        id: changes
+        with:
+          base: 'prod'
```

Comment:
I don't understand base and ref are the same. how can be there changes?

Reply:
[From the action README](https://github.com/dorny/paths-filter): When `base` and `ref` are the same, changes are detected against the most recent commit before the push. I tried using `prod` and `prod~1` first, but that doesn't work

![image](https://user-images.githubusercontent.com/1158666/130245291-87672723-767b-4e16-9c8d-eee3f30af70d.png)